diff --git a/CMakeLists.txt b/CMakeLists.txt
index fa28a6e890..8e0db028c0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,7 +145,7 @@ set_package_properties(Plasma PROPERTIES
     TYPE RUNTIME
 )
 
-find_package(KDecoration2 ${PROJECT_VERSION} CONFIG)
+find_package(KDecoration2 REQUIRED ${PROJECT_VERSION} CONFIG)
 set_package_properties(KDecoration2 PROPERTIES
     TYPE REQUIRED
     PURPOSE "Required for server side decoration support"
@@ -494,3 +494,5 @@ ecm_install_configured_files(INPUT plasma-kwin_x11.service.in plasma-kwin_waylan
             DESTINATION ${KDE_INSTALL_SYSTEMDUSERUNITDIR})
 
 ki18n_install(po)
+
+feature_summary(WHAT ALL INCLUDE_QUIET_PACKAGES FATAL_ON_MISSING_REQUIRED_PACKAGES)
\ No newline at end of file
