diff --git a/src/qmlcontrols/graphicaleffects/CMakeLists.txt b/src/qmlcontrols/graphicaleffects/CMakeLists.txt
index 6b4b2c34..b5ebca26 100644
--- a/src/qmlcontrols/graphicaleffects/CMakeLists.txt
+++ b/src/qmlcontrols/graphicaleffects/CMakeLists.txt
@@ -8,16 +8,4 @@ ecm_add_qml_module(graphicaleffects URI org.kde.graphicaleffects
     GENERATE_PLUGIN_SOURCE
 )
 
-qt_add_shaders(graphicaleffects "graphicaleffects_shaders"
-    BATCHABLE
-    PRECOMPILE
-    OPTIMIZED
-    PREFIX
-        "/shaders"
-    FILES
-       "badge.frag"
-       "preserveaspect.vert"
-       "lanczos2sharp.frag"
-)
-
 ecm_finalize_qml_module(graphicaleffects)
