diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index de8c77c..e14e7b9 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -99,28 +99,6 @@ else()
     set(_extra_args OPTIMIZED)
 endif()
 
-qt6_add_shaders(QuickCharts "shaders"
-    BATCHABLE
-    ZORDER_LOC 6
-    PRECOMPILE
-    PREFIX "/qt/qml/org/kde/quickcharts/shaders/"
-    FILES
-        shaders/barchart.vert
-        shaders/barchart.frag
-        shaders/linechart.vert
-        shaders/linechart.frag
-        shaders/piechart.vert
-        shaders/piechart.frag
-    OUTPUTS
-        barchart.vert.qsb
-        barchart.frag.qsb
-        linechart.vert.qsb
-        linechart.frag.qsb
-        piechart.vert.qsb
-        piechart.frag.qsb
-    ${_extra_args}
-)
-
 ecm_generate_export_header(QuickCharts
     BASE_NAME QuickCharts
     GROUP_BASE_NAME KF
