diff --git a/src/client/plasmawindowmanagement.cpp b/src/client/plasmawindowmanagement.cpp
index eb1621a..1186798 100644
--- a/src/client/plasmawindowmanagement.cpp
+++ b/src/client/plasmawindowmanagement.cpp
@@ -55,7 +55,7 @@ private:
     static void stackingOrderDone(void *data, struct org_kde_plasma_stacking_order *org_kde_plasma_stacking_order);
 
     static struct org_kde_plasma_window_management_listener s_listener;
-    static const org_kde_plasma_stacking_order_listener s_stackingOrderListener;
+    static struct org_kde_plasma_stacking_order_listener s_stackingOrderListener;
     PlasmaWindowManagement *q;
 };
 
